### PR TITLE
Document Fast-track process

### PR DESCRIPTION
**Pull request by llvm-beanz**
This updates the process docs to add a new documented "fast-track" process for introducing conforming extensions to HLSL.

This new process is intended to make it easy to quickly add vendor or platform specific extensions as long as they conform to the existing language definition.

### CODE REVIEW

**Review by damyanp — APPROVED** [review]
Seems like the templates could do with updating, or maybe we need a language extension template?

File: proposals/templates/extension-template.md
Comment (damyanp) [inline proposals/templates/extension-template.md:20]: Any value in an EXT prefix or something to make them stand out from language proposals?

File: docs/Process.md
Comment (s-perron) [inline docs/Process.md:11]:
```suggestion
2. Some HLSL features may instead go through a

```

**Review by s-perron — COMMENTED** [review]
LGTM, but I'll wait to see if I hear from anyone else in the Vulkan community before I approve.

File: docs/Process.md
Comment (bogner) [inline docs/Process.md:163]: Technically `[NodeIsProgramEntry]` wouldn't be valid since it isn't namespaced, but I still think it's a good example. Can we word this to reinforce that?

**Review by bogner — APPROVED** [review]
Overall I think this sounds good.

File: proposals/templates/extension-template.md
Comment (llvm-beanz) [reply proposals/templates/extension-template.md:20]: My gut says for the proposals maybe not, but I'd like us to create a directory structure of extension specifications where we do something like what Khronos does where we have a platform/vendor prefix that also gets assigned.

**Review by s-perron — APPROVED** [review]

**Comment by s-perron** [comment]
@llvm-beanz Is there anything hold this back? I want to start the HLSL spec of cooperatative matrices in Vulkan using this.

**Comment by llvm-beanz** [comment]
> @llvm-beanz Is there anything hold this back? I want to start the HLSL spec of cooperatative matrices in Vulkan using this.

Nope! I had wanted some feedback from other people internally and they privately gave me their approval so I'll merge this.

**Pull request merged**
